### ISSUE TITLE
Client access token and subscription key are embedded in the code

### ISSUE THREAD

**Issue by vik-y** [adityathiru/Bro#1]
In the file [bro.py](https://github.com/adityathiru/Bro/blob/master/bro.py) Access Token and Subscription code for AI api is embedded into the code. You should not make this public and ask users to use their own access token and keys while using this code. 

Maybe you can make a separate configuration file to store access tokens and not commit that to your code.

**Comment by adityathiru** [comment]
You are right bro, should have kept the keys in a seperate file and shouldn't have committed them. I simply left it there since the idea is in a very primitive stage and I was testing frequently before. I'll remove the keys from the config file in the next commit. Thanks ! :)
